mri/operators/fourier/orc_wrapper.py

```diff
@@ -0,0 +1,154 @@
+                 coefficients="svd", weights="full", L="auto", n_bins="auto"):
+        """ Initialize and compute multi-linear correction coefficients
+
+        Parameters
+        ----------
+        fourier_op: OperatorBase
+            Cartesian or non-Cartesian Fourier operator to wrap
+        field_map: numpy.ndarray
+            B0 field inhomogeneity map (in Hz)
+        time_vec: numpy.ndarray
+            1D vector indicating time after pulse in each shot (in s)
+        mask: numpy.ndarray
+            Mask describing the regions to consider during correction
+        coefficients: {'svd', 'mti', 'mfi'}, optional
+            Type of interpolation coefficients to use (default is 'svd')
+        weights: {'full', 'sqrt', 'log', 'ones'}
+            Weightning policy for the field map histogram (default is "full")
+        L: int, "auto"
+            Number of interpolators used for multi-linear correction
+        n_bins: int, "auto"
+            Number of bins for the field map histogram
+        """
+
+        # Redirect fourier_op essential variables
+        self.fourier_op = fourier_op
+        self.shape = fourier_op.shape
+        self.samples = fourier_op.samples
+        self.n_coils = fourier_op.n_coils
+
+        # Initialize default values
+        range_w = (np.min(field_map), np.max(field_map))
+        if (L == "auto"):
+            L = int(np.around((range_w[1] - range_w[0]) / 30))
+        if (n_bins == "auto"):
+            n_bins = 2 * int(np.around(range_w[1] - range_w[0]))
+
+        # Initialize wrapper variables
+        self.mask = mask
+        self.time_vec = time_vec
+        self.n_bins = n_bins
+        self.L = L
+
+        # Define coefficient policies
+        self.coefficients = coefficients
+        self.weights = weights
+        if (coefficients == "svd"):
+            self.compute_coefficients = compute_svd_coefficients
+        elif (coefficients == "fsvd"):
+            self.compute_coefficients = compute_fsvd_coefficients
+        elif (coefficients == "mfi"):
+            self.compute_coefficients = compute_mfi_coefficients
+        elif (coefficients == "mti"):
+            self.compute_coefficients = compute_mti_coefficients
+        else:
+            raise NotImplementedError(
+                "Unknown B0 correction coefficients: {}".format(coefficients))
+
+        # Prepare indices to reformat C from E=BC
+        self.field_map = field_map
+        scale = (range_w[1] - range_w[0]) / self.n_bins
+        scale = scale if (scale != 0) else 1
+        self.indices = np.around((field_map - range_w[0]) / scale).astype(int)
+        self.indices = np.clip(self.indices, 0, self.n_bins - 1)
+
+        # Compute the E=BC factorization and reformat B
+        self.B, self.C, self.E = self.compute_coefficients(field_map, time_vec,
+                                                           mask, L, weights,
+                                                           n_bins)
```

Comment:
```suggestion
        self.B, self.C, self.E = self.compute_coefficients(
            field_map, 
            time_vec,
            mask, 
            L, 
            weights,
            n_bins,
        )
```

I feel this is cleaner